queries.sql

```diff
@@ -7,4 +7,45 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals 
+WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) AS no_escapes FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals WHERE escape_attempts = 0;
+
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
```

Comment:
- [ ] Hi there! 👋 In this query, the goal is to fetch the animal with the highest number of escape attempts. 🏃‍♂️ Currently, it seems like the query is returning the sum. Please adjust it to retrieve the maximum escape attempt value for more accurate results. 📈💡 Precision matters! Thanks for making this tweak! 👍

```suggestion
 SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```